package.json

```diff
@@ -63,6 +68,7 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-ga": "2.7.0",
+    "regenerator-runtime": "^0.13.3",
```

Comment:
0.x version, but also: where did this come from?

Reply:
See previous comment